server/odc-service/src/main/java/com/oceanbase/odc/service/pldebug/session/DebuggeeSession.java

```diff
@@ -85,10 +85,10 @@ public DebuggeeSession(ConnectionSession connectionSession, ThreadPoolExecutor d
                     debugId = resultSet.getString(1);
                 }
             }
-            // 打开调试开关
-            stmt.execute("call dbms_debug.debug_on();");
             // 打开pl的日志输出
             enableDbmsOutput(stmt);
+            // 打开调试开关
```

Comment:
Why the problem will be solved by adjusting the execution sequence?

Reply:
if you execute `dbms_debug.debug_on` which means that you will hang up in the next sql you executed. ob-server will consider that the next sql you executed is the target that you want to debug.

so that I swap them to avoid the situation I mentioned above.

you may be interested in the reason why does these codes work if any sql executed after `dbms_debug.debug_on` will hang up, I just want to say I don't know either😂